code/modules/client/preferences_toggles.dm

```diff
@@ -396,6 +397,15 @@
 			prefs.save_preferences()
 			return "Off"
 
+/client/proc/toggle_admin_sound_types()
+	var/result = tgui_alert(src, "Which sound type do you want to toggle?", "Toggle MIDI/Internet sound type to play", list("Meme", "Atmospheric"))
+	if(result == "Meme")
+		prefs.toggles_sound ^= SOUND_ADMIN_MEME
+		to_chat(src, SPAN_NOTICE("You will [prefs.toggles_sound & SOUND_ADMIN_MEME ? "now" : "no longer"] hear meme admin sounds."))
+	if(result == "Atmospheric")
+		prefs.toggles_sound ^= SOUND_ADMIN_ATMOSPHERIC
+		to_chat(src, SPAN_NOTICE("You will [prefs.toggles_sound & SOUND_ADMIN_ATMOSPHERIC ? "now" : "no longer"] hear atmospheric admin sounds."))
+
```

Comment:
make the list strings meme [enabled] etc so people can see what they have turned on and off

Reply:
done